gram.y

```diff
@@ -478,6 +483,96 @@ v6addrlist_rasrcaddress	: IPV6ADDR ';'
 		}
 		;
 
+nat64prefixdef	: nat64prefixhead optional_nat64prefixplist ';'
+		{
+			if (nat64prefix) {
+
+				if (nat64prefix->AdvValidLifetime > 65528)
+				{
+					flog(LOG_ERR, "AdvValidLifeTime must be "
+						"smaller or equal to 65528 in %s, line %d",
```

Comment:
consume the value from the constant definition

Reply:
Done